_po/ja/how-to/laravel.po

```diff
@@ -252,13 +353,26 @@ msgid ""
 "};\n"
 "```"
 msgstr ""
+"    /**\n"
+"     * Reverse the migrations.\n"
+"     */\n"
+"    public function down(): void\n"
+"    {\n"
+"        Schema::dropIfExists('posts');\n"
+"    }\n"
+"};\n"
+"```"
 
 msgid ""
 "use Illuminate\\Database\\Migrations\\Migration;\n"
 "use Illuminate\\Database\\Schema\\Blueprint;\n"
 "use Illuminate\\Support\\Facades\\DB;  // Don't forget to add this\n"
 "use Illuminate\\Support\\Facades\\Schema;"
 msgstr ""
+"use Illuminate\\Database\\Migrations\\Migration;\n"
+"use Illuminate\\Database\\Schema\\Blueprint;\n"
+"use Illuminate\\Support\\Facades\\DB;  // Don't forget to add this\n"
+"use Illuminate\\Support\\Facades\\Schema;"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.